policyengine_uk/data/datasets/frs/calibration/calibrate.py

```diff
@@ -337,7 +337,7 @@ def generate_model_variables(dataset: str, time_period: str = "2023") -> Tuple:
         "total_national_insurance",
         "employment_income",
         "self_employment_income",
-        "pension_income",
+        "private_pension_income",
```

Comment:
does it matter for the data processing that we used to calibrate for all of pension income and now we change it to private pensions?

Reply:
Not really, we just have two names for the same thing- I'll adjust